### PR TITLE
create

### PR DESCRIPTION
**Pull request by AbenezerAssefa**
Portfolio

Built With
Major languages
                  -   HTML
                  -   CSS
Getting Started
   I created a mobile version of Portfolio

👤 Author
        GitHub: [@githubhandle](https://github.com/AbenezerAssefa)
        LinkedIn: [LinkedIn](https://www.linkedin.com/in/abenezer-assefa-386b25193)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -121,456 +122,11 @@ nav button {
 
 .illustration {
   width: 150px;
-  height: 430px;
+  height: 930px;
   background-repeat: no-repeat;
   background-image: url('../images/illustration-header-mobile.png');
```
Comment (PinkMoon25) [inline css/style.css:127]: Please fix the position of the background image to match the figma design :point_down: 
image
your design :point_down: 
image
You can try adding `background-position: bottom; `

File: css/style.css [hunk 2]
```diff
@@ -46,7 +47,7 @@ nav button {
 .home-page {
   padding: 0 15px;
   background: #1c1a19;
-  height: 590px;
+  height: 690px;
   padding-right: 24px;
 }
```
Comment (PinkMoon25) [inline css/style.css:53]: Please use the correct font based on figma template for your title , your font family is roboto :point_down: 
image
the figma design font-family is Crete Round :point_down: 
image
our goal here is to make our site as close to the original design as possible

File: css/style.css [hunk 3]
```diff
@@ -46,7 +47,7 @@ nav button {
...
```
Comment (PinkMoon25) [inline css/style.css:52]: Please use flexbox to align this section content, as mentioned in the project requiirements _**"In order to lay out the elements on the page you should use Flexbox in all 2 sections"**_ 
our goal here is to master [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and become a flex ninja  :ninja: as it is a crucial skill in front-end development

**Review by PinkMoon25 — COMMENTED** [review]
Hi @AbenezerAssefa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linters checks :heavy_check_mark: 
Professional readme :heavy_check_mark: 
follows github flow :heavy_check_mark: 
follows figma design template :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[OPTIONAL] Please use meaningful PR title which describe an action for example Setup_toolbar_and_headline
[OPTIONAL] Pleaase use meaningful pull request description which describes actions or tasks you have done in the project such as 
- Add toolbar, 
- Add headline Section, 
- Use flex for section layout

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbenezerAssefa** [comment]
Portfolio

Built With
Major languages
- HTML
- CSS
Getting Started
I created a mobile version of Portfolio

👤 Author
GitHub: [@githubhandle](https://github.com/AbenezerAssefa)
[LinkedIn](https://www.linkedin.com/in/abenezer-assefa-386b25193): LinkedIn

**Pull request closed without merging**